source/js/image-wall.bindings.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file
+ * Javascript binding of macy.js.
+ */
+(function (Drupal) {
+  'use strict';
+
+  Drupal.behaviors.gentBaseLoadImageWall = {
+    attach: function (context, settings) {
+
+      if (!Macy) { // eslint-disable-line no-undef
+        return;
+      }
+
+      var list = document.getElementsByClassName('image-wall');
```

Comment:
Why is this?
Does Macy break if the query returns no results?

Reply:
Damn, that's... bad code.
I'm sure someone will create an issue in the library :innocent: